src/test/java/fr/inria/coming/spoon/repairability/RepairabilityTest.java

```diff
@@ -24,6 +24,11 @@ public void testRepairabilityInterface() throws Exception {
         TestUtills.runRepairability("JMutRepair", "/repairability_test_files/one/");
     }
 
+    @Test
+    public void testDiffResults() throws Exception {
+        TestUtills.runRepairabilityPrint("ALL", "/dataset/commons-lang/");
```

Comment:
`/dataset/commons-lang/` will not pass the CI test since this repo is present just on your local machine right now and hence an error will be thrown.

Make it `repogit4testv0` maybe.